### DISCUSSION TITLE
duda REPORTE-PLANETAS

### DISCUSSION THREAD

**Discussion by CarlangaUC** [IIC2133-PUC/2024-1#35] [Tarea 0] [answered]
Que pasa si mi output da bien en el sentido que esta descendente pero en caso de empate no me da igual que el que aparece de output corrrecto, me explico:

```python
...
Planeta 1: 3
Planeta 2: 3
...

```
y mi caso:

```python
...
Planeta 2: 3
Planeta 1: 3
...

Estaria bien de igual modo? Como evaluan estos casos bordes?

**Comment by PabloNavarroY** [comment, marked as answer]
Esta duda ya fué respondida en #23. Se imprime primero el planeta con menor id.
